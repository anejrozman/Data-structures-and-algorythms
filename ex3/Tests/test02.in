8,13,4
5.11147930142607,2.4319836751633392,5.80390342518859,4.577157130743366,0.8722996955569624,1.7633351404777868,4.177357639009828,1.6572342474361232
0,1,388.012
3,0,276.213
3,7,668.804
0,2,156.033
3,1,44.356
6,7,57.225
5,3,960.725
4,2,474.066
2,6,482.241
2,7,738.52
6,0,370.479
4,6,115.411
6,1,631.703
