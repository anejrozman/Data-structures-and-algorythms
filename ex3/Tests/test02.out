3,12.0124
2,29.6665
2,11.1316
1,0
